### PR TITLE
Updating pcb_assembly instructions standoff ref designators

### PR DESCRIPTION
**Pull request by ericjunkins**
Updating reference designators for the standoffs in pcb assembly. Replacing PR #200 because of migration from Latex -> Markdown

### CODE REVIEW

File: electrical/pcb_assembly/README.md [hunk 1]
```diff
@@ -264,7 +264,7 @@ On the **BOTTOM** side of the board, attach
 #### 3.9.2 RoboClaw mounting Standoffs
 
 On the **BOTTOM** side of the board, attach
-    the twenty #4-40 Standoffs **T5** using screws **B8** as shown below. You
+    the twenty #4-40 Standoffs **T5* using screws **B8** as shown below. You
```
Comment (apollokit) [inline electrical/pcb_assembly/README.md:267]:
```suggestion
    the twenty #4-40 Standoffs **T5** using screws **B8** as shown below. You
```

restoring to bold

**Review by apollokit — DISMISSED** [review]
One tiny change requested, otherwise looks good

Not looking forward to updating this document on the next pcb rev...

**Review by Achllle — APPROVED** [review]
LGTM

**Comment by ericjunkins** [comment]
@apollokit can you green light this as well?

**Pull request merged**